Bottom: 4a2a615817b3d249d54a9821bb401af6f136f16d
Top:    c9a943dcf902b0bbe9c8e99e542b8779d61c38f2
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-02 16:02:03 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/include/automount.h b/include/automount.h
index 210712d..46e8530 100644
--- a/include/automount.h
+++ b/include/automount.h
@@ -80,7 +80,7 @@ int load_autofs4_module(void);
 #define MS_UNBINDABLE (1 << 17)
 #endif
 
-unsigned long mountflags = MS_UNBINDABLE|MS_MGC_VAL;
+extern unsigned long mountflags;
 
 /* This sould be enough for at least 20 host aliases */
 #define HOST_ENT_BUF_SIZE	2048
diff --git a/lib/mounts.c b/lib/mounts.c
index d7b1fe7..27cf08b 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -45,6 +45,8 @@ static const char mnt_name_template[]      = "automount(pid%u)";
 static struct kernel_mod_version kver = {0, 0};
 static const char kver_options_template[]  = "fd=%d,pgrp=%u,minproto=3,maxproto=5";
 
+unsigned long mountflags = MS_UNBINDABLE|MS_MGC_VAL;
+
 unsigned int linux_version_code(void)
 {
 	struct utsname my_utsname;
